src/main/resources/application.properties

```diff
@@ -11,4 +11,6 @@ rdf4j.repositoryUrl=http://localhost:8080/rdf4j-server/repositories
 rdf4j.workbenchUrlRepository=http://localhost:8080/rdf4j-workbench/repositories/s-pipes-hello-world
 rdf4j.repositoryName=s-pipes-hello-world
 rdf4j.pConfigURL=../s-pipes/doc/examples/hello-world/config.ttl
-rdf4j.driver=cz.cvut.kbss.ontodriver.sesame.SesameDataSource
\ No newline at end of file
+rdf4j.driver=cz.cvut.kbss.ontodriver.sesame.SesameDataSource
+
+cors.allowedOrigins: http://localhost:4173, http://localhost:5173
```

Comment:
```suggestion

cors.allowedOrigins: http://localhost:4173, http://localhost:5173
```

Reply:
@palagdan 
This is useful only for developement. For now I would like to keep development options out of default configuration if possible ... but we will see, maybe i will reconsider it later, if this cause too much pain for new developers.

Reply:
let me know if u disagree...